keyboards/adkb96/Readme.md

```diff
@@ -0,0 +1,17 @@
+# ADKB96
```

Comment:
Could you please rename this file to a fully-lowercase filename?

e.g. `keyboards/adkb96/readme.md`

Reply:
Thank you,I changed its name.